client/state/data-layer/wpcom/sites/automated-transfer/eligibility/index.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import { get } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	AUTOMATED_TRANSFER_ELIGIBILITY_REQUEST,
+} from 'state/action-types';
+
+import { updateEligibility } from 'state/automated-transfer/actions';
+import {
+	AUTOMATED_TRANSFER_STATUS,
+	eligibilityHolds,
+} from 'state/automated-transfer/constants';
+
+const eligibilityUrl = domain =>
+	`https://public-api.wordpress.com/rest/v1/sites/${ domain }/automated-transfers/eligibility`;
+
+const statusMapping = {
```

Comment:
the reason I put this here is because the `ELIGIBILITY_HOLDS` are states in Calypso and the API potentially has altogether different constants. I wanted this explicit mapping step so that we break the coupling to the API in this single file